packages/shared/friends/sagas.ts

```diff
@@ -757,6 +770,14 @@ export async function getFriendRequestsProtocol(request: GetFriendRequestsPayloa
     // Get friends
     const friends: FriendsState = getPrivateMessaging(store.getState())
 
+    // Reject blocked users
+    const blockedUsers = await handleBlockedUsers(friends.fromFriendRequests)
+    blockedUsers
+      .filter((blockedUser) => blockedUser.error !== null)
```

Comment:
I know this is currently `null` when errors are reported, but in any case a more generic check might be better here (both `null` and `undefined` would be catched).
```suggestion
      .filter((blockedUser) => blockedUser.error)
```